addons/l10n_us_account/models/res_company.py

```diff
@@ -0,0 +1,13 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    def _compute_show_invoice_tax(self):
+        super()._compute_show_invoice_tax()
+        for record in self:
+            if record.account_fiscal_country_id.code == 'US':
```

Comment:
We don't want to show these taxes only when using avatax, not always.
Or am I missing something?

That's even the reason why it is for US and BR...

Reply:
that's a good question! nothing is said in the task about it being only for Avatax users, but you might be right. @chklop what do you say?

Reply:
Note that I already mentioned it here https://github.com/odoo/odoo/pull/162347#discussion_r1598521205

Reply:
So in the US, as in Brazil, if you want to add all your taxes on the lines, you always have many to apply. In the US, you'll have County Taxes, State Taxes, Federal Taxes, etc etc. Having them mentionned, and splitted on Invoices, is not the usual practice over there.

It's the reason why, on many US invoice templates I've found, they don't have a "tax" column on them. Because it's a mess and it's not useful, "Taxes" are just mentionned at the bottom, grouped, and nobody cares about the column and the detail.

In l10n_br, they've already removed the column from all templates altogether (including Sale Orders, website, etc), and now they want to do that in the US as well. You'll see in the chatter of the task that I challenged the idea with Catalina, our US PO from San Francisco, and they still think we should go with it.

Reply:
I believe that that is a very Avatax approach.
Yes, in theory you have these taxes. But if you only have a local business, you can use simple taxes most of the time.

Reply:
Hi! As I've mentioned before it is not a common practice (nor a federal requirement) to have a tax column in customer-facing documents showing per-line taxes in US invoices. Even when doing benchmark you ca see that the most common behavior is to show the tax breakdown after subtotal.
I understand this is a big change, so I'll take some days to do some last research and decide based on the overall conclusion for the most common practice.

Reply:
Hello everyone, it is okay on our side to follow the approach of hiding the taxes only if Avatax is being used. Although it is common for companies in the US not to show per-line taxes (it is not a requirement) smaller companies or some corner cases can accept seeing those taxes. 

Let me know when the changes are done so I can do the functional testing.

Thanks! 

cc @ren-odoo